### PR TITLE
External posts in blog

### PR DESCRIPTION
**Pull request by johanandren**
As decided during meeting in Stockholm.

 * External blog/presentations moved to blog posts, this means they will automatically
   end up in news feed on landing page
 * Re-marked all posts with one of three categories: news, article, external
 * 5 latest for each of the three categories and an archive page for each
 * harmonized tags across posts
 * tags page cover all so moved up top in blog right hand nav bar

### CODE REVIEW

**Comment by johanandren** [comment]
Screenshot 2020-03-13 at 12 06 00
Screenshot 2020-03-13 at 12 04 51
Screenshot 2020-03-13 at 12 04 28

**Comment by johanandren** [comment]
Looks like I messed up while copying entries, please hold on reviewing for a bit

File: blog/_posts/2019-05-15-external-reactive-integration-akka-streams-and-alpakka.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+author: Jacek Kunicki
+category: external
+tags: [alpakka, streams, videos]
+---
+
+External presentation: 
+
+> An alpaca is like a camel, but is Alpakka like Apache Camel? Can we use the reactive approach in the area of integration as well, instead of the blocking approach that Camel usually offers? In this live coding session we’re going to explore the possibilities that Alpakka - a set of connectors for Akka Streams - offers to fetch data from various data sources and send it to different destinations. Using those, you’re going to learn, on a working example, how to build a reactive integration layer with the help of Akka Streams - one of the most popular Reactive Streams implementations. And if you thought Akka and friends were only for the Scala people, come to learn the nice Java APIs and see how wrong you were.
+
+See the whole presentation at: [https://www.youtube.com/watch?v=gUNiEZG8cbM](https://www.youtube.com/watch?v=gUNiEZG8cbM)
```
Comment (patriknw) [inline blog/_posts/2019-05-15-external-reactive-integration-akka-streams-and-alpakka.md]: should the link text be the URL?

File: blog/_posts/2019-07-11-external-tour-of-akka-typed.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+author: Manuel Bernhardt
+category: external
+tags: [actors, typed]
+---
+
+External blog article series: 
+
+> In this series we are going to explore Akka Typed, the new Akka Actor API that brings significant advantage over the classic one. If you’re not familiar with the Akka Actor API, don’t worry — this series is intended to be understandable even if you do not. If you are familiar with the Actor API then this series will help you understand and learn how to work with Akka Typed.
+
+Read the whole article series at: [https://manuel.bernhardt.io/articles/#akka-typed](https://manuel.bernhardt.io/articles/#akka-typed)
```
Comment (patriknw) [inline blog/_posts/2019-07-11-external-tour-of-akka-typed.md]: link text vs url? Read the whole article series at [Manuel's blog].

File: blog/_posts/2019-05-15-external-reactive-integration-akka-streams-and-alpakka.md (hunk 1 above)
Comment (johanandren) [reply blog/_posts/2019-05-15-external-reactive-integration-akka-streams-and-alpakka.md]: I think it is nice to show off the full URL we send people when it is an external, and just putting the url in there didn't make jekyll turn it into a link for some reason.

**Pull request merged**